ESP32/config/credentials.h.template

```diff
@@ -0,0 +1,13 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#pragma once
```

Comment:
I would suggest using only include guards throughout the whole project, or if we decide to go with both at least go with 'pragma once' first and then the include guards.

Reply:
In the current hsb project only ifndef is used, so I would also be fine with using only this too. But most of the time I have also seen "pragma once" after ifndef. And I also think it's auto generated this way if you create a class directly with vs code (plugins)

Reply:
For me I just want to prevent that the header is loaded twice and both should achieve that, so I am fine with all options.
So what's the final verdict? just use `#pragma once` without the include guards?

Reply:
Keep it as is here, I'll make a separate issue for this.